t/16-utils.t

```diff
@@ -341,25 +341,32 @@ subtest 'base_host' => sub {
 subtest 'project directory functions' => sub {
     local $ENV{OPENQA_BASEDIR};
     local $ENV{OPENQA_SHAREDIR};
-    is prjdir(),    '/var/lib/openqa',               'right directory';
-    is sharedir(),  '/var/lib/openqa/share',         'right directory';
-    is resultdir(), '/var/lib/openqa/testresults',   'right directory';
-    is assetdir(),  '/var/lib/openqa/share/factory', 'right directory';
-    is imagesdir(), '/var/lib/openqa/images',        'right directory';
-
+    is prjdir(),     '/var/lib/openqa',               'right directory';
+    is sharedir(),   '/var/lib/openqa/share',         'right directory';
+    is resultdir(),  '/var/lib/openqa/testresults',   'right directory';
+    is assetdir(),   '/var/lib/openqa/share/factory', 'right directory';
+    is imagesdir(),  '/var/lib/openqa/images',        'right directory';
+    is gitrepodir(), '',                              'not url when distri is not defined';
     local $ENV{OPENQA_BASEDIR} = '/tmp/test';
     is prjdir(),    '/tmp/test/openqa',               'right directory';
     is sharedir(),  '/tmp/test/openqa/share',         'right directory';
     is resultdir(), '/tmp/test/openqa/testresults',   'right directory';
     is assetdir(),  '/tmp/test/openqa/share/factory', 'right directory';
     is imagesdir(), '/tmp/test/openqa/images',        'right directory';
-
+    my $distri = 'opensuse';
+    is gitrepodir(distri => $distri), '', 'empty when .git is missing';
+    my $mocked_git = path(sharedir . "/tests/opensuse/.git")->make_path;
+    $mocked_git->child('config')
+      ->spurt(qq{[remote "origin"]\n        url = git\@github.com:b10n1k/os-autoinst-distri-opensuse.git});
```

Comment:
better avoid personal avatar names in the openQA source code. And please don't hardcode to os-autoinst-distri-opensuse, how about github.com/os-autoinst/os-autoinst-distri-example ?

Reply:
*this change is ready for review*